test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Waiters.cs

```diff
@@ -44,6 +44,8 @@ public static void WaitForFileToBeReadable(string file, TimeSpan timeout)
 
         public static void WaitForProcessToStop(int processId, TimeSpan timeout, bool expectedToStop, string errorMessage)
         {
+            Console.WriteLine($"Waiting for process {processId} to stop...");
```

Comment:
That line is very useful, otherwise you see the test doing nothing for a few seconds and then it fails. Then you don't know if it failed because the timeout was wrong or because it really timedout